coderd/audit/request.go

```diff
@@ -71,6 +71,8 @@ func ResourceTarget[T Auditable](tgt T) string {
 	case database.APIKey:
 		// this isn't used
 		return ""
+	case database.License:
+		return typed.UUID.String()
```

Comment:
I think the numerical ID might make more sense as the target, rather than just having the UUID for both.

```suggestion
		return strconv.Itoa(typed.ID)
```